setup.py

```diff
@@ -35,19 +34,8 @@
         ),
     ],
     install_requires=[
-        "click==8.1.3",
-        "h5py",
-        "microdf_python",
-        "pandas",
-        "pathlib",
         "policyengine-core",
-        "pytest",
-        "pytest-dependency",
         "pyyaml",
```

Comment:
we can remove this too - while it's not in -core, it's not imported anywhere

Reply:
OK